### PR TITLE
Documentation

### LINKED ISSUE #3: Add documentation to GitHub
Includes:
- meeting minutes

### PR DESCRIPTION
**Pull request by AA666666**

### CODE REVIEW

**Comment by JadenBalogh** [comment]
See issue #3

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Home Together
 Home Together is an online hub for all things home-sharing.
 
+Documentation:
```
Comment (JadenBalogh) [inline README.md]:
```suggestion
## Documentation
```

**Review by JadenBalogh — CHANGES_REQUESTED** [review]
See comments for changes.

**Review by JadenBalogh — APPROVED** [review]
Looks good.

**Review by aKrisnanto — APPROVED** [review]
All good.

**Pull request merged**